.ci/test_lint_doctests.py

```diff
@@ -81,6 +82,36 @@ def test_docker_build_matrix():
         assert existing_build_matrix == f.read()
 
 
+@pytest.mark.timeout(30)
+def test_json_schemas():
+    """Test JSON Schemas are up to date."""
+    schemas_folder = pathlib.Path(os.path.dirname(__file__)) / '..' / 'composer' / 'json_schemas'
+
+    # Capture existing schemas
+    existing_schemas = []
+    for filename in sorted(os.listdir(schemas_folder)):
+        if filename != 'generate_json_schemas.py':
+            with open(os.path.join(schemas_folder, filename), 'r') as f:
+                existing_schemas.append(json.load(f))
+
+    # Run the script
+    check_output(
+        subprocess.run(['python', 'generate_json_schemas.py'], cwd=schemas_folder, capture_output=True, text=True))
+
+    def sort_json(json):
```

Comment:
nit: can you move this to a parent function? not a fan of functions in functions.